auth/server/src/main/java/com/seagates3/controller/AccountController.java

```diff
@@ -99,8 +100,26 @@ public ServerResponse create() {
         String name = requestBody.get("AccountName");
         String email = requestBody.get("Email");
         Account account;
-
+        int accountCount = 0;
         LOGGER.info("Creating account: " + name);
+        int maxAccountLimit = AuthServerConfig.getMaxAccountLimit();
+        int internalAccountCount =
+            AuthServerConfig.getS3InternalAccounts().size();
+        int maxAllowedLdapResults = maxAccountLimit + internalAccountCount;
+
+        try {
+          accountCount = accountDao.getTotalCountOfAccounts();
+
+          if (accountCount >= maxAllowedLdapResults) {
```

Comment:
this check should be added in ListAccounts?....are we just throwing error if accounts exeed this amount without listing any account or after listing 1000 accounts

Reply:
we already have this check for FetchAll() method for listing accounts and there we are fetching records till <maxlimit> + internal account count  and next records will be ignored.